### PR TITLE
Make chinese README better

### PR DESCRIPTION
**Pull request by ischaojie**
I read the Chinese version of README, it has a much of grammatical errors and punctuation errors.

Meanwhile, I update it to the latest README file.

(By the way, I'm Chinese.)

### CODE REVIEW

File: README_chinese.md [hunk 1]
```diff
@@ -60,85 +58,88 @@ $ pip install 'httpx[cli]'  # 只安装集成的命令行功能
...
-
-加上这些应该具备的标准功能
-
-* 域名与URL
-* 保持活动状态
-* 具有Cookie持久性的会话
-* 浏览器风格的SSL验证
-* 身份验证
-* Elegant Key/Value Cookies
+HTTPX 建立在成熟的请求可用性基础上，为您提供以下功能：
```
Comment (abersheeran) [inline README_chinese.md:61]: “请求”应该改成`requests`，保持包名不变。

File: README_chinese.md [hunk 2]
```diff
@@ -60,85 +58,88 @@ $ pip install 'httpx[cli]'  # 只安装集成的命令行功能
...
+
+* 广泛的 [请求兼容 API](https://www.python-httpx.org/compatibility/)。
```
Comment (abersheeran) [inline README_chinese.md:63]: 同上，“兼容`requests`”

File: README_chinese.md [hunk 3]
```diff
@@ -60,85 +58,88 @@ $ pip install 'httpx[cli]'  # 只安装集成的命令行功能
...
+* 国际化域名与 URL
+* Keep-Alive & 连接池
+* Cookie 持久性会话
+* 浏览器风格的 SSL 验证
+* 基础或摘要身份验证
+* 优雅的键值 Cookies
 * 自动解压缩
-* 内容自动解码
-* Unicode Response Bodies
+* 自动内容解码
```
Comment (abersheeran) [inline README_chinese.md:81]: “内容自动解码”或者“自动的内容解码”会更通顺一些。

File: README_chinese.md [hunk 4]
```diff
@@ -60,85 +58,88 @@ $ pip install 'httpx[cli]'  # 只安装集成的命令行功能
...
+* Unicode 响应正文
 * 分段文件上传
-* 支持HTTP(S) 代理
-* 支持设定timeout
+* HTTP(S) 代理支持
```
Comment (abersheeran) [inline README_chinese.md:84]: 符号后无需空格。

File: README_chinese.md [hunk 5]
```diff
@@ -60,85 +58,88 @@ $ pip install 'httpx[cli]'  # 只安装集成的命令行功能
...
+* 连接超时
```
Comment (abersheeran) [inline README_chinese.md:85]: “可配置连接超时”更好一些，会没那么突兀。

**Comment by Kludex** [comment]
Once again... Thanks @abersheeran 🙏

**Comment by abersheeran** [comment]
Since @ischaojie and I are both Chinese, and this is a PR for Chinese documents, in order to facilitate communication, I will directly use Chinese to propose revisions.

File: README_chinese.md (hunk 5 above)
Comment (ischaojie) [reply README_chinese.md:85]: “连接超时处理” 呢

**Review by abersheeran — APPROVED** [review]

**Pull request merged**